addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CodeInjectionScanRule.java

```diff
@@ -272,9 +272,8 @@ private boolean testAspInjection(String paramName) {
                 }
 
                 // Check if the injected content has been evaluated and printed
-                if (msg.getResponseBody()
-                        .toString()
-                        .contains(Integer.toString(bignum1 * bignum2))) {
+                String evidence = Integer.toString(bignum1 * bignum2);
```

Comment:
I am not sure if the author wanted to do the integer overflow or not so I kept the previous code, but I would use longs. I have no experience with asp.